### PR TITLE
Added Dart Online Compiler #30daysofappwrite Project

### PR DESCRIPTION
**Pull request by aadarshadhakalg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 <!-- Raise a PR and add a link to your project under this section. -->
 * [Title of your Project](http://link_to_your_project)
```
[the remark this answers is not in the document]
Comment (aadarshadhakalg) [reply README.md:55]: done

[the remark this answers is not in the document]
Comment (gameatronist) [reply README.md:55]: can I know the language that you're writing code?

**Comment by gameatronist** [comment]
Thank you

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aadarshadhakalg) [reply README.md:55]: it's [dart](https://dart.dev/)

[the remark this answers is not in the document]
Comment (gameatronist) [reply README.md:55]: ok aadarshadhakalg

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**